File: 2778-sum-of-squares-of-special-elements/2778-sum-of-squares-of-special-elements.py
class Solution:
    def sumOfSquares(self, nums: List[int]) -> int:
        ans = 0

        for i in range(1, len(nums) + 1):
            if len(nums) % i == 0:
                ans = ans + (nums[i - 1] ** 2)

        return ans

File: 2778-sum-of-squares-of-special-elements/2778-sum-of-squares-of-special-elements.cpp
class Solution {
public:
    int sumOfSquares(vector<int>& nums) {
        int ans = 0;
        for (int i = 1; i <= nums.size(); i++) {
            if (nums.size() % i == 0) ans = ans + (nums[i - 1] * nums[i - 1]);
        }
        return ans;
    }
};